lib/managementClient.ts

```diff
@@ -77,37 +129,401 @@ export class ManagementClient extends LinkEntity {
+        message_id: generate_uuid(),
+        reply_to: this.replyTo,
+        application_properties: {
+          operation: Constants.operations.cancelScheduledMessage
+        }
+      };
+      request.application_properties![Constants.trackingId] = generate_uuid();
+      log.mgmt("Cancel scheduled messages request body: %O.", request.body);
+      log.mgmt("[%s] Acquiring lock to get the management req res link.",
+        this._context.namespace.connectionId);
+      await defaultLock.acquire(this.managementLock, () => { return this._init(); });
+      await this._mgmtReqResLink!.sendRequest(request);
+    } catch (err) {
+      const error = translate(err);
+      log.error("An error occurred while sending the request to cancel the scheduled message to " +
+        "$management endpoint: %O", error);
+      throw error;
+    }
+  }
+
+  /**
+   * Receives a specific deferred message identified by `sequenceNumber` of the `Message`.
+   * @param sequenceNumber The sequence number of the message that will be received.
+   * @param receiveMode The mode in which the receiver was created.
+   * @returns Promise<ServiceBusMessage | undefined>
+   * - Returns `ServiceBusMessage` identified by sequence number.
+   * - Returns `undefined` if no such message is found.
+   * - Throws an error if the message has not been deferred.
+   */
+  async receiveDeferredMessage(sequenceNumber: Long, receiveMode: ReceiveMode): Promise<ServiceBusMessage | undefined> {
+    if (!Long.isLong(sequenceNumber)) {
+      throw new Error("'sequenceNumber' is a required parameter and must be an instance of 'Long'.");
+    }
+    let message: ServiceBusMessage | undefined = undefined;
+    const messages = await this.receiveDeferredMessages([sequenceNumber], receiveMode);
+    if (messages.length) {
+      message = messages[0];
+    }
+    return message;
+  }
+
+  /**
+   * Receives a list of deferred messages identified by `sequenceNumbers`.
+   * @param sequenceNumbers A list containing the sequence numbers to receive.
+   * @param receiveMode The mode in which the receiver was created.
+   * @returns Promise<ServiceBusMessage[]>
+   * - Returns a list of messages identified by the given sequenceNumbers.
+   * - Returns an empty list if no messages are found.
+   * - Throws an error if the messages have not been deferred.
+   */
+  async receiveDeferredMessages(sequenceNumbers: Long[], receiveMode: ReceiveMode): Promise<ServiceBusMessage[]> {
+    if (!Array.isArray(sequenceNumbers)) {
+      throw new Error("'sequenceNumbers' is a required parameter and must be of type 'Array'.");
+    }
+
+    if (typeof receiveMode !== 'number') {
+      throw new Error("'receiveMode' is a required parameter with value 1 or 2.");
+    }
+
+    const messageList: ServiceBusMessage[] = [];
+
+    try {
+      const messageBody: any = {};
+      messageBody["sequence-numbers"] = types.wrap_array(sequenceNumbers.map((i) => { return Buffer.from(i.toBytesBE()); }), 0x81, undefined);
```

Comment:
nit: you can use the arrow function shorthand here

```suggestion
      messageBody["sequence-numbers"] = types.wrap_array(sequenceNumbers.map((i) => Buffer.from(i.toBytesBE())), 0x81, undefined);
```

There's also a [tslint rule](https://palantir.github.io/tslint/rules/arrow-return-shorthand/) if you want it automatically enforced